### PR TITLE
[15.0][MIG] hr_attendance_rfid: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Superseed https://github.com/OCA/hr-attendance/pull/66

Migration to 15.0

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT36462

### CODE REVIEW

File: hr_attendance_rfid/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "HR Attendance RFID",
-    "version": "14.0.1.1.0",
+    "version": "15.0.1.1.0",
```
Comment (CarlosRoca13) [inline hr_attendance_rfid/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Review by CarlosRoca13 — CHANGES_REQUESTED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by lubusax** [comment]
I already improved the codecov using this test: 
(https://github.com/lubusax/hr-attendance/blob/74724f9ca1c65907c656827650c361cd224fb9a4/hr_attendance_rfid/tests/test_hr_attendance_rfid_process.py#L59)
I launched a PR on the same subject after you. Mine has been closed, but maybe you could reuse this test.

**Comment by victoralmau** [comment]
> 

Ok, i have added the tests you indicate in a separate commit.

**Comment by nimarosa** [comment]
Everyone it's okay with this migration?

**Comment by pedrobaeza** [comment]
@nimarosa why don't you do a proper review?

**Comment by nimarosa** [comment]
@pedrobaeza for me it's okay as this migration don't change any of the previous versión code. 

But as it has not been merged I wanted to know if anyone has any comment to do. 

I will test it in my instance today to be sure and we are ready. Also it's not much I can test since I have not a rfid device available to test. But will test installing and functional.

**Comment by lubusax** [comment]
> @pedrobaeza for me it's okay as this migration don't change any of the previous versión code.
> 
> But as it has not been merged I wanted to know if anyone has any comment to do.
> 
> I will test it in my instance today to be sure and we are ready. Also it's not much I can test since I have not a rfid device available to test. But will test installing and functional.

Hi, I have installed the module and tried it on some RFID Attendance devices (I build and sell them myself). Works like a charm.

**Pull request merged**